README.md

```diff
@@ -345,47 +350,73 @@ Name        | Value example | Description
-`p_import`      | W       | Power Import
-`p_import_react` | VAR    | Reactive Power Import
-`p_import_apparent` | VA  | Apparent Power Import
-`p_import_avg`  | W       | Power Import average
-`p_import_min`  | W       | Power Import minimum that day
-`p_import_max`  | W       | Power Import max that day
-`p_export`      | W       | Power Export
-`p_export_react` | VAR    | Reactive Power Export
-`p_export_min`  | W       | Power Export minimum that day
-`p_export_max`  | W       | Power Export max that day
-`p_factor`      |         | Power Factor
+`e_import`      | kWh     | Energy import
+`e_export`      | kWh     | Energy export
+`last_e_export` | kWh     | Energy export that day
+`last_e_import` | kWh     | Energy import that day
+`p_import`      | W       | Power import
+`p_import_react` | VAR    | Reactive power import
+`p_import_apparent` | VA  | Apparent power import
+`p_import_avg`  | W       | Power import average
+`p_import_min`  | W       | Power import min that day
+`p_import_max`  | W       | Power import max that day
+`p_export`      | W       | Power export
+`p_export_react` | VAR    | Reactive power export
+`p_export_min`  | W       | Power export min that day
+`p_export_max`  | W       | Power export max that day
+`p_factor`      | -       | Power factor
 `freq`          | Hz      | Frequency
-`freq_min`      | Hz      | Frequency Min
-`freq_max`      | Hz      | Frequency Max
+`freq_min`      | Hz      | Frequency min
+`freq_max`      | Hz      | Frequency max
 `u1`            | V       | Voltage phase 1
 `u2`            | V       | Voltage phase 2
 `u3`            | V       | Voltage phase 3
 `i1`            | A       | Current phase 1
 `i2`            | A       | Current phase 2
 `i3`            | A       | Current phase 3
-`dc_p`          | W       | DC Power
-`dc_p_min`      | W       | DC Power
-`dc_p_max`      | W       | DC Power
-`dc_u`          | V       | DC Voltage
-`dc_u_min`      | V       | DC Min Voltage
-`dc_u_max`      | V       | DC Max Voltage
-`dc_i`          | A       | DC Current
-`dc_i_min`      | A       | DC Min Current
-`dc_i_max`      | A       | DC Max Current
+`dc_p`          | W       | DC power
+`dc_p_min`      | W       | DC min power
+`dc_p_max`      | W       | DC max power
+`dc_u`          | V       | DC voltage
+`dc_u_min`      | V       | DC min voltage
+`dc_u_max`      | V       | DC max voltage
+`dc_i`          | A       | DC current
+`dc_i_min`      | A       | DC min current
+`dc_i_max`      | A       | DC max current
+
+#### Examples
+
+Meter reading reported by a energy meter:
+```json
+{
+   "type": "evt.meter.report",
+   "serv": "meter_elec",
+   "val_t": "float",
+   "val": { 123.5 },
+   "prop": {"unit": "kWh", "delta_t": 3600},
+}
+```
 
+Extended meter reading reported by a one phase AC current meter:
+```json
+{
+   "type": "evt.meter_ext.report",
+   "serv": "meter_elec",
+   "val_t": "float_map",
+   "val": { 
+    "u1": 237,
+    "i1": 0.12,
+    "p_import": 28.4,
+    },
```

Comment:
imo You shall put props in the example, but make it empty(?)

Reply:
@boguslaw-wojcik ? Is props field mandatory, even if empty ? We should state it somwhere if true.